Assets/BossRoom/Scripts/Client/UI/IPUIMediator.cs

```diff
@@ -87,9 +75,14 @@ public void HostIPRequest(string ip, string port)
 
             m_GameNetPortal.PlayerName = m_PlayerNameLabel.text;
 
-            m_GameNetPortal.StartHost(ip, portNum);
-
-            m_SignInSpinner.SetActive(true);
+            if (m_GameNetPortal.StartHost(ip, portNum))
+            {
+                m_SignInSpinner.SetActive(true);
+            }
+            else
+            {
+                PopupManager.ShowPopupPanel("Connection Failed", "Server failed to bind.");
```

Comment:
can a different issue cause StartHost to return false?

Reply:
Regardless of the reason, the error returned by UTP is the same, so I think it makes sense to log this generic message.